packages/webpack-config/babel.config.js

```diff
@@ -0,0 +1,13 @@
+// babel.config.js
```

Comment:
i think all the jest-related stuff should be located in the root dir, to be later extended by the packages

Reply:
https://github.com/facebook/jest/issues/3112